Cargo.toml

```diff
@@ -34,7 +34,7 @@ reqwest = "0.9.5"
 rocket = "0.4.0"
 rocket_contrib = "0.4.0"
 uuid = { version = "0.7", features = ["v4"] }
-rust-crypto = "^0.2"
+rust-crypto = { git = "https://github.com/vhnatyk/rust-crypto", branch="aarch64"} #"^0.2"
```

Comment:
what is the change between the "official" rust-crypto and yours? 
lets talk about this point because it is problematic in terms of upgradability

Reply:
I forked where author implemented method `rust_crypto_util_fixed_time_eq_asm` that was missing in original [rust-crypto](https://github.com/DaGenix/rust-crypto) for arm64 - as rust-crypto is not actively developed (perhaps due to being quite feature-complete?), those changes were not merged to master. The method is not used neither in core of gg18, not in core of lindell2017, but in benches for lindell2017 - so it's an open question is it required. I had to fork it,  but maybe it's a good idea for KZen to fork it as rust-crypto isn't actively changing anymore anyway (just like you forked some other libs). The only chages from original `rust-crypto` **master** is that it `Adds a definition of rust_crypto_util_fixed_time_eq_asm for ARMv8. It's
exactly the same as for ARMv7, except that it uses ARMv8 names for
registers (wN instead of rN)` Looks to me like there are no good or obvious reasons why it wasn't merged to master of rust-crypto. @omershlo What do you think?

Reply:
I use rust_crypto only for GG18 with network (in the bin folder). It is suppose to be possible to run the test without this library...
having said that - did you send a PR for rust_crypto?
If we couldn't make it go away we will probably fork it as you suggest.

Reply:
Original PR from the branch that I forked is hanging there for quite a while(few years) - no point to do another PR with same changes I think, as rust_crypto seems not maintained anymore (looks like it was split into separate libraries that got abandoned too). Let me rethink a bit with this, I'm working with Android right now, and the changes don't compile there ok with GCC so I'm trying clang - but I found another repo, where author fixed for both iOS and Android for aarch64 and it compiles ok for both platforms. At the moment I'm analyzing those changes

Reply:
Imho fork by KZen is the best option - changes for compatibility with aarch64 to rust_crypto_util_fixed_time_eq_asm are relatively minor and seem to look solid, hmm